OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj

```diff
@@ -1,32 +1,32 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
-    <Description>Server part of M2M support for OpenRiaServices. It includes Code generator and Fluent configuration extensions for OpenServices.M2M.</Description>
-    <PackageId>OpenRiaServices.M2M</PackageId>
-  </PropertyGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="System.ComponentModel.Annotations" Version="4.5.0" />
-  </ItemGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net472' ">
-    <Reference Include="System" />
-    <Reference Include="System.ComponentModel.Composition" />
-    <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Runtime.Serialization" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="EntityFramework" Version="6.2.0" />
-    <PackageReference Include="OpenRiaServices.FluentMetadata" Version="5.0.0-rc.1" />
-    <PackageReference Include="OpenRiaServices.Server" Version="5.0.0-rc0001" />
-    <PackageReference Include="OpenRiaServices.T4" Version="5.0.0-rc0001" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\OpenRiaServices.M2M.LinkTable\OpenRiaServices.M2M.LinkTable.csproj" />
-  </ItemGroup>
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472</TargetFrameworks>
+    <Description>Server part of M2M support for OpenRiaServices. It includes Code generator and Fluent configuration extensions for OpenServices.M2M.</Description>

```

Comment:
Why does the whole file show up as a diff? Cannot se on mobile

Reply:
seems to be some kind of whitespace diff

Reply:
After further investigation it seems like the original file only used linefeed endings
I assume we want windows standard CR LF ?

Reply:
Cr lf should be fine, git should convert it to lf on check in given standard gitattribute file

You might want to check that there is a default gitattribute file in the root such as https://github.com/dotnet/core/blob/master/.gitattributes

Reply:
Done